README.rst

```diff
@@ -52,8 +56,8 @@ The easiest way to install aioriak is by using the package on PyPi::
 Requirements
 ------------
 
-- Python >= 3.5
-- riak>=2.1.3
+- Python >= 3.9
+- riak>=2.7.0
```

Comment:
We should probably mention v3.0.0 from https://github.com/people-ai/riak-python-client/pull/1/files, right?

Reply:
fixed